db/migrate/20230118100324_add_deleted_at_to_billable_metrics.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+class AddDeletedAtToBillableMetrics < ActiveRecord::Migration[7.0]
+  def change
+    add_column :billable_metrics, :deleted_at, :datetime
```

Comment:
`index: true` does not work here?

Reply:
Nope - https://apidock.com/rails/ActiveRecord/ConnectionAdapters/SchemaStatements/add_column